django/db/backends/mysql/features.py

```diff
@@ -30,6 +30,7 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     supports_select_intersection = False
     supports_select_difference = False
     supports_slicing_ordering_in_compound = True
+    strindex_matches_case_insensitively = True
```

Comment:
But MySQL is case insensitive for all string comparisons by default, and there isn't a database feature for that 🤔